x-pack/plugins/enterprise_search/public/applications/esre/components/esre_guide/esre_guide.tsx

```diff
@@ -7,23 +7,84 @@
 
 import React from 'react';
 
+import {
+  EuiImage,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiHorizontalRule,
+} from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
 
+import analyticsIllustration from '../../../../assets/images/analytics.svg';
+import scalableIllustration from '../../../../assets/images/scalable.svg';
+import simplifyIllustration from '../../../../assets/images/simplify.svg';
 import { SetEsreChrome as SetPageChrome } from '../../../shared/kibana_chrome';
 import { EnterpriseSearchEsrePageTemplate } from '../layout/page_template';
 
+import { EsreDocsSection } from './esre_docs_section';
+import { MeasurePerformanceSection } from './measure_performance_section';
+import { RankAggregationSection } from './rank_aggregation_section';
+import { SemanticSearchSection } from './semantic_search_section';
+
 export const EsreGuide: React.FC = () => {
```

Comment:
I'm not sure what logic I should cover with unit tests; everything in this PR is using EUI components. Maybe I can add a test that `<EsreGuide>` renders the custom sections. Thoughts?

Reply:
I think you're good for now. We will sometimes test components with enzyme, but usually only if there is complex logic controlling rendering.